docs/rules/accessor-pairs.md

```diff
@@ -273,14 +273,15 @@ var o = {
 
 The code above creates an object with just a setter for the property `"a"`.
 
-See [no-dupe-keys](no-dupe-keys.md) if you also want to disallow duplicate keys in object literals.
-
-See [no-dupe-class-members](no-dupe-class-members.md) if you also want to disallow duplicate names in class definitions.
-
```

Comment:
I think this should stay here, as it's in the context of the limitations explained in this section.

Reply:
Changed.